packages/template-retail-react-app/app/pages/cart/index.jsx

```diff
@@ -534,13 +600,32 @@ const Cart = () => {
                 <Stack spacing={24}>
                     <Stack spacing={4}>
                         <CartTitle />
-
                         <Grid
                             templateColumns={{base: '1fr', lg: '66% 1fr'}}
                             gap={{base: 10, xl: 20}}
                         >
                             <GridItem>
                                 <Stack spacing={4}>
+                                    <Box layerStyle="cardBordered" p={3}>
+                                        {isPickupOrder ? (
+                                            <Text fontWeight="bold">
+                                                <FormattedMessage
+                                                    defaultMessage="Pickup in Store ({storeName})"
+                                                    id="cart.order_type.pickup_in_store"
+                                                    values={{
+                                                        storeName
+                                                    }}
+                                                />
+                                            </Text>
+                                        ) : (
+                                            <Text fontWeight="bold">
+                                                <FormattedMessage
+                                                    defaultMessage="Delivery"
+                                                    id="cart.order_type.delivery"
+                                                />
+                                            </Text>
+                                        )}
+                                    </Box>
```

Comment:
Preferably separate this to a component of its own.

BTW, I am seeing just a div with the word "delivery" on top when my cart has no pickup item. That seems a bit out of place.